assets/css/dashboard/new-pa-message-page/associate-alert-page.scss

```diff
@@ -0,0 +1,201 @@
+.associate-alert-page {
+  padding: 24px 48px;
+}
+
+.associate-alert-page-header {
+  &__cancel {
+    flex: none;
+    width: 100px;
+
+    button {
+      margin: 8px 16px 8px 16px;
+      height: 48px;
+      color: $text-link-primary;
+      text-decoration: none;
+
+      &:hover {
+        color: $text-link-primary;
+        background-color: $cool-gray-15;
+      }
+    }
+  }
+}
+
+.associate-alert-page-body {
+  flex-wrap: nowrap;
+}
+
+.associate-alert-filter-selection {
+  flex: none;
+  font-size: 20px;
+  width: 200px;
+  padding: 0px 12px;
+  margin-top: 40px;
+
+  &__button-group {
+    margin-bottom: 48px;
+  }
+
+  &__selected {
+    color: #ffffff !important;
+    background: #d9d9d93d !important;
+  }
+
+  button {
+    text-align: left;
+    height: 38px;
+    min-width: 200px;
+    color: $text-secondary;
+    background-color: $cool-gray-20;
+    border: none;
+  }
+}
+
+.associate-alert-table {
+  margin: 40px 48px;
```

Comment:
For some reason, it seems like the page is ignoring the right margin here. Table is overflowing out of the `col` and out of the viewport. If you remove the `margin-right` here and make it `padding-right` in `.associate-alert-page-body` it might be a bit more what you are looking for.

Minor note: As I build out some of the PA Message components, I'm coming to the conclusion that the Bootstrap magic in the `Row` and `Col` components are just frustrating to work with. On the Select Stations page, I started using `div`s in a flexbox and find that it's much easier to get margins and paddings to work how I expect them to. Thinking a bit into the future, I wonder if we should do our own layouts and just use Bootstrap for the visual components. Nothing to do here, just thinking out loud.

Reply:
Ah yep that does look better to me, thank you!

And yeah I was kind of running into those struggles at first as well. I think eventually I just sort of got used to it and worked it out but I'm onboard for avoiding using `Row` and `Col` moving forward